warehouse/templates/pages/security-key-giveaway.html

```diff
@@ -0,0 +1,205 @@
+        </p>
+      {% elif not request.user %}
+        {# The user is logged out, ask them to log in #}
+        <a href="{{ request.route_url("accounts.login", _query={REDIRECT_FIELD_NAME: request.path_qs}) }}" class="button button--primary">
+          Log in to determine if you are eligible
+        </a>
+      {% elif not eligible %}
+        {# The user is not eligible, tell them why #}
+        <p>Sorry, you're not currently eligible for a promo code.</p>
+        <p><b>Reason:</b> {{ reason_ineligible }}</p>
+      {% else %}
+        {# The user is eligible, show the form #}
+        <p>
+          Congrats, you may be eligible to receive a promo code! The following projects you maintain are considered critical projects:
+        </p>
+        <ul>
+          {% for project in eligible_projects %}
+          <li>{{ project }}</li>
+          {% endfor %}
+        </ul>
+        <p>
+          We just need a little more information. (<a href="#regions">Why?</a>)
+          <form method="POST">
+            <input name="csrf_token" type="hidden" value="{{ request.session.get_csrf_token() }}">
+            {{ form.country }}
+            <button type="submit" class="button button--primary">Request a promo code</button>
+          </form>
+        </p>
+      {% endif %}
+      </div>
+
+
+      <section class="faq-group">
+      <h2 id="faq">FAQ</h2>
+      <p>Answers to frequently asked questions regarding this effort:</p>
+
+      <h3 id="critical-eligiblity">What determines if project is a critical project?</h3>
+      <p>
+        PyPI determines project eligibility based on download counts derived
+        from PyPI's <a
+        href="https://warehouse.pypa.io/api-reference/bigquery-datasets.html">public
+      dataset of download statistics</a>. Any project in the top 1% of
+    downloads over the prior 6 months is designated as critical.
+      </p>
+
+      <h3 id="critical-quantity">How many projects are designated as critical projects?</h3>
+      <p>
+        At the time of writing, there are more than 350K projects on PyPI,
+        resulting in a 'critical' designation of more than 3,500 projects. This
+        determination is recalculated on a daily basis.
+      </p>
+
+      <h3 id="optout">Can a project opt-out or become non-critical in any way?</h3>
+      <p>
+        No, once the project has been designated as critical it retains that
+        designation indefinitely.
+      </p>
+
+      <h3 id="user-requirement-eligiblity">What users are included in the 2FA requirement?</h3>
+      <p>
+        Any maintainer of a critical project (both 'Maintainers' and 'Owners')
+        are included in the 2FA requirement.
+      </p>
+
+      <h3 id="user-key-eligibility">What users are eligible to receive security keys?</h3>
+      <p>
+        PyPI users who are eligible to receive security keys must be
+        maintainers of critical projects who have not previously enabled 2FA on
```

Comment:
I'd really like to encourage security keys over TOTP. could we exclude TOTP as a 2FA method from consideration and extend these to any maintainer who is not currently using WebAuthn?

Reply:
I think if we do that we would need to revisit whether 1% is the right % for critical project designation, or whether we're OK with running out of promo codes.

Right now for the top 1%:
* 3853 total maintainers:
  * 2759 do not not have 2FA enabled
  * 1094 have some 2FA enabled:
    * 894 have only TOTP enabled
    * 113 have both TOTP and security keys
    * 87 have only security keys
    
So that change would take the # of eligible users from 2.7K to 3.6K, which means roughly half of them would not be able to get a promo code.

Reply:
I'm _very_ happy to see that coverage as it is! OK in that case, let's stick with initial plan for now to get greatest coverage.

Reply:
I think we can probably ldo some auxiliary efforts (informational callouts, etc) to generally motivate the TOTP-only users to adopt hardware keys.